crates/bevy_app/src/schedule_runner.rs

```diff
@@ -123,7 +123,7 @@ impl Plugin for ScheduleRunnerPlugin {
                     {
                         fn set_timeout(f: &Closure<dyn FnMut()>, dur: Duration) {
                             web_sys::window()
-                                .unwrap()
+                                .expect("Could not get window.")
```

Comment:
this probably can only fail when running the wasm not in a browser, it would be helpful to mention

Reply:
Does this sound appropriate?
`A browser is required to run wasm, but none was found`